scripts/zones/Northern_San_dOria/npcs/Charlaimagnat.lua

```diff
@@ -32,7 +33,21 @@ end;
 -----------------------------------
 
 function onTrigger(player,npc)
-	player:startEvent(0x02be);
+
+	local realday = tonumber(tostring(os.date("%Y")) .. os.date("%m") .. os.date("%d"));
+	local TheMissingPiece = player:getQuestStatus(OUTLANDS,THE_MISSING_PIECE);
+	
+	if(TheMissingPiece == QUEST_ACCEPTED and player:hasKeyItem(TABLET_OF_ANCIENT_MAGIC) and player:hasKeyItem(LETTER_FROM_ALFESAR)) then
+		player:startEvent(0x02bf);--Continuing the Quest
+		player:setVar("TheMissingPiece_date", 0);

```

Comment:
i suppose it was left behind when i was coding the quest to start. i was just trying to figure out how setVar worked. Is there a way to delete vars after they've been used? I dont think anyone will be repeating this quest after its completed.